app/views/static_pages/community_rules.html.erb

```diff
@@ -13,166 +13,151 @@
-        <strong>Mild toxicity</strong> such as treating others on the server disrespectfully, directed insults, provoking/baiting other users, passive aggression, naming & shaming, trolling, or other unprofessional behavior.
-      </li>
-      <li>
-        <strong>Posting self promotions without permission</strong> by sharing resources you created solely for personal or monetary gain, for example: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (could mislead learners), "Why PHP is superior" (unrelated blog post), job recruitment. Instead, <strong>send a DM to ModMail</strong> to ask for permission before posting.
-      </li>
-    </ul>
-
-    <h3 id="zap-level-5">5 Zaps -  Warning for High Severity Violations</h3>
-
-    <ul>
-      <li>
-        <strong>Posting about any illegal activities</strong>, including any drug-related conversations.
-      </li>
-      <li>
-        <strong>Publicly arguing about receiving moderation activity after being asked to take it to ModMail/discuss privately</strong>
-      </li>
-      <li>
-        <strong>Excessive toxicity</strong> such as targeted harassment, inciting drama, aggressively arguing purposefully, etc.
-      </li>
-      <li>
-        <strong>Plagiarism</strong> such as stealing other learners' work
-      </li>
-    </ul>
-
-    <h3 id="zap-level-10">10 Zaps - Immediate Ban for Extreme Severity Violations</h3>
-
-    <ul>
-      <li>
-        <strong>Piracy</strong>
-      </li>
-      <li>
-        <strong>Doxxing</strong>
-      </li>
-      <li>
-        <strong>Continued harassment</strong>
-      </li>
-      <li>
-        <strong>NSFW or otherwise highly offensive imagery in posts, profile pictures, or usernames</strong>
-      </li>
-      <li>
-        <strong>Bigotry</strong> such as racism, homophobia, hate speech, etc.
-      </li>
-      <li>
-        <strong>Spamming posts</strong> such as Discord Nitro spam, repeated random nonsensical messages (copypasta), or self promotion (joining just to promote material, no prior chat history).
-      </li>
-      <li>
-        <strong>Any other behavior deemed unacceptable and deserving of a ban by the moderation team</strong> such as not correcting <a href="https://slash7.com/2006/12/22/vampires/" target="_blank">“help vampire”</a> behaviors.
-      </li>
-    </ul>
-
-    <h3 id="decay-example">Decay Explanation</h3>
-
+    <h3 id="safe-topics">Discuss topics that are safe and welcoming for everyone</h3>
     <p>
-      Zaps decay at a rate of 1 zap per week. E.g., if you receive 5 zaps on a Friday, the following Friday you will have 4, the Friday after that you will have 3, and so forth. This means higher level infractions remain on record for a longer period of time and jeopardizes your participation in the server if rules continue to be broken.
+      <span class="fa-regular fa-circle-check mr-1"></span> Discuss topics that are relevant to our curriculum or have an off-topic channel.
```

Comment:
The smaller icons don't need as much, just a little extra spacing and the same colors as the title icons:
```suggestion
      <span class="fa-regular fa-circle-check mr-2 text-green-700"></span> Discuss topics that are relevant to our curriculum or have an off-topic channel.
```

Outputs:
<img width="845" alt="Screenshot 2023-03-28 at 09 03 08" src="https://user-images.githubusercontent.com/7963776/228170124-b70ceb67-bb43-4991-9143-9bcb326f185d.png">